rubygem-scaptimony/rubygem-scaptimony.spec

```diff
@@ -0,0 +1,109 @@
+# This package contains macros that provide functionality relating to
+# Software Collections. These macros are not used in default
+# Fedora builds, and should not be blindly copied or enabled.
+# Specifically, the "scl" macro must not be defined in official Fedora
+# builds. For more information, see:
+# http://docs.fedoraproject.org/en-US/Fedora_Contributor_Documentation/1/html/Software_Collections_Guide/index.html
+
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name scaptimony
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.2.0
+Release: 1%{?dist}
+Summary: SCAPtimony is SCAP database and storage server
+Group: Applications/System
+License: GPLv3
+URL: https://github.com/OpenSCAP/scaptimony
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+%if 0%{?fedora} > 18
+Requires: %{?scl_prefix}ruby(release)
+Requires: %{?scl_prefix}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(openscap) >= 0.4.0
+Requires: %{?scl_prefix}rubygem(rails) >= 3.2.8
```

Comment:
Seems to be in one if/else block, and not the other?  (Maybe just de-dupe it and move outside the conditional.)